packages/client/src/getForeignCoins.ts

```diff
@@ -0,0 +1,9 @@
+import { ZetaChainClient } from "./client";
+
+export const getForeignCoins = async function (this: ZetaChainClient) {
+  const api = await this.getEndpoint("cosmos-http", `zeta_${this.network}`);
+  const endpoint = `${api}/zeta-chain/fungible/foreign_coins`;
+  const response = await fetch(endpoint);
+  const data = await response.json();
+  return data.foreignCoins;
```

Comment:
I agree, we need to fix the types we generate and publish from the node. Right now they're unusable, because they don't match the format returned by the API: https://github.com/zeta-chain/node/issues/1795